buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -858,9 +860,10 @@ class BeamModulePlugin implements Plugin<Project> {
       // configurations because they are never required to be shaded or become a
       // dependency of the output.
       def compileOnlyAnnotationDeps = [
-        "com.google.auto.value:auto-value-annotations:1.7",
+        "com.google.auto.value:auto-value-annotations:$autovalue_version",
```

Comment:
Nit: can you insert the new lines according to alphabetical order?

Reply:
(I realize this one wasn't in exact alphabetical order to begin with, but it should have been.)